README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Add a description of what 'go-task' is and why it's being used to improve clarity for users unfamiliar with it.